### ISSUE TITLE
Low fragment number in several 5degC samples 

### ISSUE THREAD

**Issue by laurahspencer** [RobertsLab/resources#1904]
I'm analyzing the RNASeq data that I aligned with STAR/featureCounts, and notice that several samples from the 5degC treatment (blue) have many fewer fragments (aligned/assigned to genes). @shedurkin do you see that in your analysis? 

![image](https://github.com/RobertsLab/project-cod-temperature/assets/17264765/db5071ac-bb99-4488-b44f-914e3eef2e64)
![image](https://github.com/RobertsLab/project-cod-temperature/assets/17264765/e0618f59-8a64-4191-a79d-d002da48dfda)

**Comment by laurahspencer** [comment]
The plot thickens... 
There are also some alignment read depth differences among treatments in the low coverage Whole Genome Sequencing data, particularly in the 5C treatment. 

![image](https://github.com/RobertsLab/resources/assets/17264765/7ee11505-d048-4559-8091-1b28e4b1e8c4)
![image](https://github.com/RobertsLab/resources/assets/17264765/39a18a16-8152-45bd-b3ed-ef0dd13bb876)

**Comment by laurahspencer** [comment]
Sam- did you end up reaching out to the RNASeq sequencing facility about the weird read depth in many samples from the 5C treatment?   What was the minimum # sequences that they needed to provide?  Several samples have <10M.  Feel free to connect me with them and I can do the questioning - it'd be great to get some answers from them!  

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/LAURA~1.SPE/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/LAURA~1.SPE/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">


Sample   Name | M   Seqs
-- | --
149_R1_001 | 96.9
149_R2_001 | 96.9
58-G_R1_001 | 29.4
58-G_R2_001 | 29.4
57-G_R1_001 | 28.9
57-G_R2_001 | 28.9
150_R1_001 | 27.5
150_R2_001 | 27.5
148_R1_001 | 24.8
148_R2_001 | 24.8
19-S_R1_001 | 24.8
19-S_R2_001 | 24.8
20-G_R1_001 | 24.3
20-G_R2_001 | 24.3
127_R1_001 | 23.5
127_R2_001 | 23.5
19-G_R1_001 | 23
19-G_R2_001 | 23
108_R1_001 | 22.6
108_R2_001 | 22.6
117_R1_001 | 22.5
117_R2_001 | 22.5
119_R1_001 | 22.4
119_R2_001 | 22.4
98_R1_001 | 22.3
98_R2_001 | 22.3
109_R1_001 | 22.2
109_R2_001 | 22.2
11_R1_001 | 21.6
11_R2_001 | 21.6
121_R1_001 | 21.5
121_R2_001 | 21.5
110_R1_001 | 21.3
110_R2_001 | 21.3
1_R1_001 | 21.3
1_R2_001 | 21.3
118_R1_001 | 21.2
118_R2_001 | 21.2
4_R1_001 | 20.9
4_R2_001 | 20.9
120_R1_001 | 20.8
120_R2_001 | 20.8
12_R1_001 | 20.7
12_R2_001 | 20.7
3_R1_001 | 20.7
3_R2_001 | 20.7
67_R1_001 | 20.6
67_R2_001 | 20.6
60_R1_001 | 20.5
60_R2_001 | 20.5
68_R1_001 | 20.5
68_R2_001 | 20.5
48_R1_001 | 20.2
48_R2_001 | 20.2
49_R1_001 | 20.2
49_R2_001 | 20.2
88_R1_001 | 20.2
88_R2_001 | 20.2
91_R1_001 | 20.2
91_R2_001 | 20.2
30_R1_001 | 19.9
30_R2_001 | 19.9
57_R1_001 | 19.9
57_R2_001 | 19.9
5_R1_001 | 19.9
5_R2_001 | 19.9
10_R1_001 | 19.8
10_R2_001 | 19.8
58_R1_001 | 19.7
58_R2_001 | 19.7
2_R1_001 | 19.6
2_R2_001 | 19.6
28_R1_001 | 19
28_R2_001 | 19
31_R1_001 | 19
31_R2_001 | 19
79_R1_001 | 19
79_R2_001 | 19
29_R1_001 | 18.9
29_R2_001 | 18.9
50_R1_001 | 18.9
50_R2_001 | 18.9
59_R1_001 | 18.7
59_R2_001 | 18.7
13_R1_001 | 18.6
13_R2_001 | 18.6
47_R1_001 | 18.6
47_R2_001 | 18.6
38_R1_001 | 18.4
38_R2_001 | 18.4
78_R1_001 | 18.3
78_R2_001 | 18.3
90_R1_001 | 18.3
90_R2_001 | 18.3
39_R1_001 | 18.1
39_R2_001 | 18.1
97_R1_001 | 18.1
97_R2_001 | 18.1
100_R1_001 | 17.9
100_R2_001 | 17.9
19_R1_001 | 17.8
19_R2_001 | 17.8
18_R1_001 | 17.7
18_R2_001 | 17.7
69_R1_001 | 17.6
69_R2_001 | 17.6
21_R1_001 | 17.4
21_R2_001 | 17.4
20_R1_001 | 17.2
20_R2_001 | 17.2
107_R1_001 | 17.1
107_R2_001 | 17.1
RESUB-94_R1_001 | 16.9
RESUB-94_R2_001 | 16.9
99_R1_001 | 16.8
99_R2_001 | 16.8
40_R1_001 | 16.7
40_R2_001 | 16.7
20-S_R1_001 | 16.6
20-S_R2_001 | 16.6
83_R1_001 | 16.6
83_R2_001 | 16.6
RESUB-156_R1_001 | 16.6
RESUB-156_R2_001 | 16.6
37_R1_001 | 16.3
37_R2_001 | 16.3
41_R1_001 | 16.3
41_R2_001 | 16.3
RESUB-116_R1_001 | 16
RESUB-116_R2_001 | 16
58-S_R1_001 | 15.7
58-S_R2_001 | 15.7
70_R1_001 | 15.3
70_R2_001 | 15.3
RESUB-36_R1_001 | 15
RESUB-36_R2_001 | 15
80_R1_001 | 14.8
80_R2_001 | 14.8
RESUB-76_R1_001 | 14.8
RESUB-76_R2_001 | 14.8
57-S_R1_001 | 13.2
57-S_R2_001 | 13.2
128_R1_001 | 10
128_R2_001 | 10
139_R1_001 | 9.7
139_R2_001 | 9.7
129_R1_001 | 9.6
129_R2_001 | 9.6
140_R1_001 | 9.5
140_R2_001 | 9.5
131_R1_001 | 9.1
131_R2_001 | 9.1
147_R1_001 | 8.8
147_R2_001 | 8.8
138_R1_001 | 8.5
138_R2_001 | 8.5
137_R1_001 | 8.3
137_R2_001 | 8.3



</body>

</html>

**Comment by kubu4** [comment]
The quote for the project was ~350M paired-end reads per lane (i.e. targeting ~20M per sample).

I'll contact them regarding the disparity in output across samples.

**Comment by sr320** [comment]
Where are we with this?

**Comment by kubu4** [comment]
Azenta is currently looking into this issue.

**Comment by kubu4** [comment]
I've pinged them weekly since my last post above and finally got a response today, with an apology and a statement that they'd look into where things stand.

**Comment by laurahspencer** [comment]
Thanks for staying on top of this!

**Comment by kubu4** [comment]
They've finally responded with some info.

They've offered to "top off" the samples with an additional 20M reads. I can't see a downside to this, but figured I'd throw it out there for people to consider before I responded.

Also, a bit surprising (concerning?) is that they load samples for sequencing using volumes instead of library concentrations???!!!

Here's their response:

> Hi Sam,
>
> Thank you for your message and apologies for our delay here.
>
>One of reasons that might have led to the uneven distribution of reads among the libraries, is the inconsistent input amounts of RNA among the samples. Since the input material for the library prep for sample 149 was loaded based on a volume, as opposed to concentration, this can affect the efficiency of enzymatic reactions during prep, such as fragmentation, end-repair, and adapter ligation. Thus, even with the same preparation among all the samples, these small differences can affect cluster generation efficiency on the sequencing flow cell and ultimately lead to variations in read counts.
>
> However, I understand that the variations you are seeing here are not ideal. I am happy to top-off the samples that are below ~20M reads to provide you the minimum of quoted reads across all your samples. Would this be something of interest to you? This of course, would be done at no additional cost to you.

**Comment by laurahspencer** [comment]
Yes, that IS weird that they aren't considering concentration when prepping for sequencing. 
Getting more data for those sample with <20M reads would be great. Yes please!

**Comment by laurahspencer** [comment]
Any estimate on when they can provide the additional data?

**Comment by kubu4** [comment]
This is actually on my to-do list for today.

Data was _supposed_ to be done early last week, but haven't heard anything since.

**Comment by kubu4** [comment]
Alrighty, got the supplemental data on Friday afternoon. Download finished this weekend and I've verified MD5 checksums.

Data is here:

https://owl.fish.washington.edu/nightingales/G_macrocephalus/30-943133806/supplemental/

ATM, I've kept the original data and this supplemental sequencing separate. However, it will likely greatly simplify working with the data if I just concatenate the data so we have one set of FastQs for each sample.

Notebook is here:

https://robertslab.github.io/sams-notebook/posts/2024/2024-09-09-Data-Received---G.macrocephalus-Supplemental-RNA-seq-Data-from-Azenta-Project-30-943133806/
